docs/docs/cmd/tenant/report/report-settings-set.mdx

```diff
@@ -0,0 +1,34 @@
+import Global from '/docs/cmd/_global.mdx';
+
+# tenant report settings set
+
+Update tenant-level settings for Microsoft 365 reports
+
+## Usage
+
+```sh
+m365 tenant report settings set [options]
+```
+
+## Options
+
+```md definition-list
+`-d, --displayConcealedNames <displayConcealedNames>`
+: Determines whether all reports conceal user information such as usernames, groups, and sites. If false, all reports show identifiable information.
+
+```
+
+<Global />
+
+## Examples
+
+Set to display concealed user information in Microsoft 365 reports
+
+```sh
+m365 tenant report settings set --displayConcealedNames true
+```
+
+## Response
+
+The command won't return a response on success
+
```

Comment:
I think there's two line breaks here, let's bring that down to one. (one customary line break at the end of an md-file)